.github/workflows/check-inactive-collaborators.yml

```diff
@@ -4,6 +4,7 @@ on:
   workflow_dispatch: 
 
 env:
+    GH_TOKEN: ${{ secrets.github_ci_user_pat }}
```

Comment:
Do you specifically need to use the GitHub CI User PAT, or can you use automatic token authentication? (https://docs.github.com/en/actions/security-guides/automatic-token-authentication)

Reply:
I tried both `github.token` and `secrets.GITHUB_TOKEN`, but neither of them worked.